Orange navbar: change minimizing behavior (#1830)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/content/docs/5.3/migration.md

- **Orange footer**
  - <span class="badge bg-warning">Warning</span> For accessibility reasons, having a `aria-labelledby` on the collapse element in the accordions used in Orange footer is not necessary and can be removed. Be careful to not remove the corresponding `id` if used for other purposes. Please reflect these modifications into your websites.

- **Orange Navbar**

>   - <span class="badge bg-warning">Warning</span> The minimizing behavior with `.header-minimized` is applied on top of the `<header>` instead of one of his child directly.

Reviewer: Is it a 'warning' or a 'breaking' change?

- **Offcanvas**
  - <span class="badge bg-success">New</span> A tooltip was added on close buttons of offcanvases. Please add this tooltip in your websites, by adding the attributes `data-bs-toggle`, `data-bs-placement` and `data-bs-title`.

- **Orange Navbar**

>   - <span class="badge bg-warning">Warning</span> The minimizing behavior with `.header-minimized` is applied on top of the `<header>` instead of one of his child directly.

Reviewer 2: ```suggestion
  - <span class="badge bg-warning">Warning</span> The minimizing behavior with `.header-minimized` is applied to the `<header>` instead of one of its children directly.
```
